src/cpu.rs

```diff
@@ -527,6 +549,181 @@ impl Cpu {
         self.condition_codes.set_zero(self.registers.a);
         self.condition_codes.set_parity(self.registers.a);
     }
+
+    // arithmetic group
+    fn add(&mut self, val: u8) {
```

Comment:
Do you mind if I add the descriptions and condition codes affected from the manual to these methods? I found them helpful even when I was reviewing my functions to understand what was happening and not have to double check the documentation. You would also be able to remove some of your comments like "put result in accumulator" and "update flags" since it's already contained in the description.

Reply:
Not at all! I should have done it before. I am at work right now, hopefully I will have a little bit of downtime this afternoon were I can edit all these things. If not, I will get to it tonight.